cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/15-action-plan.html

```diff
@@ -0,0 +1,99 @@
+
+        <p>Read or ask about the requirements of each grant and scholarship. Ask if they are they renewable for following years—if not, you’ll need to find replacement funding next year. </p>
+
+        <h4>4. Make a budget.</h4>
+
+        <p>Use this cashflow budget to plan out when you’ll get money and when you’ll have to pay it.</p> 
+
+        <h4>5. Double check that you’ll have enough money when you need it.</h4>
+
+        <p>If you have uncovered direct costs, talk to the financial aid office about options to reduce the amount due at the beginning of the school year, like tuition installment plans. Or can you (or your family) start saving for these now? </p> 
+
+        <h4>6. Accept the loans you want.</h4>
+
+        <p> More paperwork ahead! You’ll also need to do entrance counseling for federal loans. Remember, you don’t need to accept all the loans you’re offered or the full amount.</p>
+
+        <h4>7. Put a reminder in your calendar to fill out FAFSA on October 1.</h4>
+
+        <p>If you’ll be in school next year, you’ll need to start this process again in a few months. </p>
+
+        <h4>8. Set another reminder to visit the financial aid office every semester.</h4>
+
+        <p>Ask about new opportunities for funding. For example, some scholarships are only open to certain majors or only for upperclassmen.</p>
+
+        <h4>9. Know how many credits you already have and make a plan to graduate on time. </h4>
+
+        <p> Make sure you get credit for your AP, IB, dual-enrollment, and transfer courses. Work with an academic adviser to stay on track for on-time graduation.</p>
+
+        <h4>10. Ask about getting credits from a more affordable school.</h4>
+
+        <p>Reach out to your academic advisor about taking core requirements at community college. Make sure they will count towards your degree. (And be strategic about when you take them – if you start school with enough credits to count as a second- or third-year student, you may be disqualified from grants and scholarships for first-year students.)</p>
+
+
+        <h4>11. If you need them, apply for additional loans.</h4>
+
+        <p>To get the best deal for your situation, read the fine print and shop around for the lowest interest rates and fees. You’ll probably need a cosigner for private loans.</p>
+
+    </div>
+
+    <div class="block block__sub action-plan_steps" data-action-plan="consider">
+        <h3>Consider other options</h3>
+
+        <h4>1. Ask about getting credits from a more affordable school.</h4>
+        <p>Reach out to your academic advisor about options like taking core requirements at community college.</p>
+
+        <h4>2. Reduce your living costs.</h4>
+        <p>Can you live at home? Stick to public transit?</p> 
+
+        <h4>3. Look for more grants and scholarships.</h4>
+        <p>There are always more available, even if school starts in a week or two! Scholarship Finder is one place to look</p>
```

Comment:
Sentence missing a period. 
Also, should we offer a link to Scholarhip Finder?

Reply:
Updated the punctuation in 9237cac; will add the link in an upcoming content update PR.